spec/features/medicaid_spec.rb

```diff
@@ -1,8 +1,142 @@
 require "rails_helper"
 
 RSpec.feature "Medicaid app" do
+  scenario "with a single member", :js do
+    visit medicaid_root_path
+
+    within(".slab--hero") do
+      click_on "Apply for Medicaid"
+    end
+
+    on_pages "Introduction" do
+      click_on "Yes"
+
+      fill_in "What is your first name?", with: "Jessie"
+      fill_in "What is your last name?", with: "Tester"
+      select_radio(question: "What is your gender?", answer: "Female")
+      click_on "Next"
+
+      expect(page).to have_content(
+        "Now tell us about any other people residing in your household.",
+      )
+      click_on "Next"
+
+      expect(page).to have_content("Are you currently a college student?")
+      click_on "No"
+
+      expect(page).to have_content("Are you currently a US Citizen?")
+      click_on "Yes"
+    end
+
+    on_pages "Health Coverage Needs" do
+      expect(page).to have_content(
+        "Are you currently enrolled in a health insurance plan?",
+      )
+      click_on "No"
+
+      expect(page).to have_content(
+        "Do you need help paying for medical expenses from the last 3 months?",
+      )
+      click_on "Yes"
+    end
+
+    on_pages "Quick Health Questions" do
+      expect(page).to have_content("Do you have a disability?")
+      click_on "No"
+
+      expect(page).to have_content("Have you been pregnant recently?")
+      click_on "No"
+    end
+
+    on_pages "Quick Tax Question" do
+      expect(page).to have_content(
+        "Do you plan on filing a federal tax return next year?",
+      )
+
+      click_on "Yes"
+    end
+
+    on_pages "Current Income" do
+      expect(page).to have_content("Do you currently have a job?")
+      click_on "Yes"
+
+      expect(page).to have_content("Tell us how many jobs you currently have.")
+      choose "1 job"
+      click_on "Next"
+
+      expect(page).to have_content("Are you self-employed?")
+      click_on "No"
```

Comment:
Don't have to do this now, but should we have a test that says "Yes" to all so that we see all possible screens?